lib/src/core/filters.dart

```diff
@@ -395,12 +395,17 @@ class _FilterEngine {
   Future uninstall(_InstantiatedFilter filter) async {
     await filter._controller.close();
     _filters.remove(filter);
-
-    if (filter.isPubSub && !_clearingBecauseSocketClosed) {
-      final connection = _client._connectWithPeer();
-      await connection?.sendRequest('eth_unsubscribe', [filter.id]);
-    } else {
-      await _rpc.call('eth_uninstallFilter', [filter.id]);
+    try {
+      if (filter.isPubSub && !_clearingBecauseSocketClosed) {
+        final connection = _client._connectWithPeer();
+        await connection?.sendRequest('eth_unsubscribe', [filter.id]);
+      } else {
+        await _rpc.call('eth_uninstallFilter', [filter.id]);
+      }
+    }
+    // ignore: avoid_catches_without_on_clauses
+    catch (e) {
+      //ignore exceptions when unregistering filters
     }
```

Comment:
I think we shouldn't ignore that lint since we can work around it more elegantly:

```suggestion
    } on Object {
      // Ignore exceptions when unregistering filters. If anything goes wrong here there's nothing
      // we could do about it either way.
    }
```

Reply:
Yes, it will be correct. But on the whole it turned out that these measures were not enough. I am still getting application stop due to exceptions when calling uninstall filter